File: Daily Pratice Question/JAVASCRIPT/question(27).js
// 27. Write a program to extract the file extension from a given filename.
// Example: Input: "document.pdf", Output: "pdf".

var arr="document.pdf"
for(i=0;i<arr.length;i++){
    if(arr[i]=="."){
        
        for(j=i+1;j<arr.length;j++){
            console.log(arr[j])
        }
    }
        
}

File: Daily Pratice Question/C LANGUAGE/Question(27).c
// 27. Write a program to extract the file extension from a given filename.
// Example: Input: "document.pdf", Output: "pdf".

#include <stdio.h>
#include<string.h>

int main() {
  char str[]="document.pdf";
 int i;
 for(i=0;i<strlen(str);i++){
     if(str[i]=='.'){
         int j;
         for(j=i+1;j<strlen(str);j++){
             printf("%c",str[j]);
         }
     }
 }

    return 0;
}